tests/IBFE/explicit_ex4_2d.inactive_1.mpirun=4.output

```diff
@@ -2177,10 +2177,10 @@ workload estimate on processor 0 = 1024
 workload estimate on processor 1 = 1024
 workload estimate on processor 2 = 1024
 workload estimate on processor 3 = 1024
-local active DoFs on processor 0 = 3222
-local active DoFs on processor 1 = 2187
-local active DoFs on processor 2 = 2187
-local active DoFs on processor 3 = 1962
+local active DoFs on processor 0 = 2655
+local active DoFs on processor 1 = 2299
+local active DoFs on processor 2 = 2361
+local active DoFs on processor 3 = 2243
```

Comment:
@abarret The large jump is a result of `LinearPartitioner` being a bad choice for getting equal numbers of degrees of freedom on each cell since it completely ignores both the geometry and topology of the mesh. The new partitioner does an OK job for quasi-uniform meshes. Thats why the differences are so large.